doc/introduction.rst

```diff
@@ -89,8 +85,72 @@ Why is machine learning relevant to NeuroImaging? A few examples!
 Installing nilearn
 ====================
 
-.. raw:: html
-   :file: install_doc_component.html
+.. tab-set::
+
+  .. tab-item:: Pip install
+
+      **1. Download and install 64-bit Anaconda**
+
+      We recommend that you install a complete
+      64-bit scientific Python distribution like
+      `Anaconda <https://www.anaconda.com/download/>`_ (or `PythonXY <http://python-xy.github.io/>`_).
+
+      Nilearn requires a Python installation and the following
```

Comment:
here we should point to the list of dependencies we already have elsewhere instead of having a copy of the list